odd_collector/adapters/redshift/mappers/tables.py

```diff
@@ -20,11 +20,13 @@ def map_table(
     mtables: MetadataTables,
     mcolumns: MetadataColumns,
     database: str,
+    schemas: List[str]
 ) -> List[DataEntity]:
     data_entities: List[DataEntity] = []
     column_index: int = 0
 
     for mtable in mtables.items:
+        if mtable.schema_name not in schemas and len(schemas): continue
```

Comment:
Odd collector has black formatter as dependency, use it with your IDE to format code